File: C/1002.c
#include <stdio.h>
#include <math.h>

#define PI 3.14159

int main()
{
    float raio = 0,
        area = 0;

    scanf(" %f", &raio);
    area = PI * pow(raio, 2);

    printf("A=%.4f\n", area);

    return 0;
}

File: C#/1002.cs
using System;
using System.Globalization;

class URI
{
    static void Main(string[] args)
    {
        double pi = 3.14159;
        double raio = double.Parse(Console.ReadLine(), CultureInfo.InvariantCulture);
        double area = Math.Round(
            pi * raio * raio,
            4
        );
        Console.WriteLine("A={0}", area.ToString("0.0000", CultureInfo.InvariantCulture));
    }
}
